src/core_plugins/kibana/public/management/sections/objects/_objects.js

```diff
@@ -205,8 +205,9 @@ uiModules.get('apps/management')
         })
         .then((overwriteAll) => {
           const conflictedIndexPatterns = [];
+          const conflictedSearchDocs = [];
 
-          function importDocument(doc) {
+          function importDocument(swallowErrors, doc) {
```

Comment:
I mentioned this to @chrisronline on Slack. The side effects of adding elements to `conflictedIndexPatterns`, and `conflictedSearchDocs` into this function makes it difficult to follow the flow of the code, especially since we call this function later and pass in `conflictedSearchDocs`.

It looks like the logic within the `if (swallowErrors ...)` conditional is the only part which changes from callsite to callsite. Can we extract this into its own function and pass that function in when we need it?